mercury/can_decoder.py

```diff
@@ -0,0 +1,46 @@
+from .can_sensors import *
+
+example_can_msg = "1000000000010000001000000010000000000000000100000000000"
+
+
+def read_bits(msg, num_bits):
+    this_value = msg[0:num_bits]
+    msg = msg[num_bits:]
+    return this_value, msg
+
+
+def decode_can_message(msg):
+    # we need the binary representation of the integer
+    msg = bin(int(msg))
+
+    msg = bin(int(example_can_msg, 2))
```

Comment:
This override of `msg` is just here so I can test with the `example_can_msg` binary string above. Will be removed before merge.